### DISCUSSION TITLE
LineValueSampler

### DISCUSSION THREAD

**Discussion by LiuPengPeter** [idaholab/moose#16366] [Q&A Modules: General] [answered]
Hello,

When I use the LineValueSampler in the s5 from phase_field/tutorials/spinodal_decomposition.

It shows that:
Task add_postprocessor is not registered to build VectorPostprocessor derived objects

I s there any problem in my change?
```
[Postprocessors]
  [./c_end]
    type =  LineValueSampler
    start_point = '0 0 0'
    end_point = '25 0 0'
    variable = c
    num_points = 26
    sort_by =  id
    execute_on = timestep_end
  [../]
[]

```

**Comment by loganharbour** [comment, marked as answer]
It’s a VectorPostprocessor, not a Postprocessor. 

Use a [VectorPostprocessors] block instead.
